src/ApiService/ApiService/onefuzzlib/ReproOperations.cs

```diff
@@ -16,7 +16,7 @@ public class ReproOperations : StatefulOrm<Repro, VmState, ReproOperations>, IRe
     private static readonly Dictionary<Os, string> DEFAULT_OS = new Dictionary<Os, string>
     {
         {Os.Linux, "Canonical:UbuntuServer:18.04-LTS:latest"},
-        {Os.Windows, "MicrosoftWindowsDesktop:Windows-10:20h2-pro:latest"}
```

Comment:
There is another issue #2225 where I'm suggesting that we remove the test, which makes these lists redundant. Further, after deployment if we still have some nodes running the older version this would cause the test to fail after this change. 

If we consider this list to be the list of fully tested images for the repro operation, then we should just add to the list.

Reply:
Understood. I can take out the change to the repro-related files.

Reply:
Honestly, this wouldn't be a bad instance_config value - i.e., make default image config values for windows and linux in the instance_config.

Reply:
+1 to migrating it to `instance_config`. Presumably, if someone chooses to use the default linux/windows they're deferring to us to make a good decision about what the image should be which includes keeping the image up to date. By adding it to the instance_config, we don't need to make code changes to update those images. When we do update them, auto scale will naturally phase the old ones out and the ones ones in.

Reply:
New issue #2229

Reply:
@mgreisen / @tevoinea - I've made an issue for a long term solution, but we need this in the short term. Please approve when you see fit.